docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -0,0 +1,206 @@
+2: enp1s0	inet 192.168.24.46/32 brd 192.168.24.255 scope global enp1s0\   	valid_lft forever preferred_lft forever
+7: br-ex	inet 10.0.0.122/24 brd 10.0.0.255 scope global br-ex\   	valid_lft forever preferred_lft forever
+8: vlan70	inet 172.17.5.22/24 brd 172.17.5.255 scope global vlan70\   	valid_lft forever preferred_lft forever
+8: vlan70	inet 172.17.5.94/32 brd 172.17.5.255 scope global vlan70\   	valid_lft forever preferred_lft forever
+9: vlan50	inet 172.17.2.140/24 brd 172.17.2.255 scope global vlan50\   	valid_lft forever preferred_lft forever
+10: vlan30	inet 172.17.3.73/24 brd 172.17.3.255 scope global vlan30\   	valid_lft forever preferred_lft forever
+10: vlan30	inet 172.17.3.68/32 brd 172.17.3.255 scope global vlan30\   	valid_lft forever preferred_lft forever
+11: vlan20	inet 172.17.1.88/24 brd 172.17.1.255 scope global vlan20\   	valid_lft forever preferred_lft forever
+12: vlan40	inet 172.17.4.24/24 brd 172.17.4.255 scope global vlan40\   	valid_lft forever preferred_lft forever
+----
++
+* vlan30 represents the Storage Network, where the new RGW instances should be
+started on the CephStorage nodes.
+* br-ex represents the External Network, which is where in the current
+environment, haproxy has the frontend VIP assigned.
+
+. Identify the network that you previously had in haproxy and propagate it through
+{OpenStackPreviousInstaller} to the CephStorage nodes. This network is used to reserve a new VIP
+that is owned by {Ceph} and used as the entry point for the RGW service.
+
+.. Log into `controller-0` and check the current HaProxy configuration until you
+find `ceph_rgw` section:
++
+----
+$ less /var/lib/config-data/puppet-generated/haproxy/etc/haproxy/haproxy.cfg
+
+...
+...
+listen ceph_rgw
+  bind 10.0.0.103:8080 transparent
+  bind 172.17.3.68:8080 transparent
+  mode http
+  balance leastconn
+  http-request set-header X-Forwarded-Proto https if { ssl_fc }
+  http-request set-header X-Forwarded-Proto http if !{ ssl_fc }
+  http-request set-header X-Forwarded-Port %[dst_port]
+  option httpchk GET /swift/healthcheck
+  option httplog
+  option forwardfor
+  server controller-0.storage.redhat.local 172.17.3.73:8080 check fall 5 inter 2000 rise 2
+  server controller-1.storage.redhat.local 172.17.3.146:8080 check fall 5 inter 2000 rise 2
+  server controller-2.storage.redhat.local 172.17.3.156:8080 check fall 5 inter 2000 rise 2
+----
+
+.. Confirm that the network is used as an HaProxy frontend:
++
+----
+[controller-0]$ ip -o -4 a
+
+...
+7: br-ex	inet 10.0.0.106/24 brd 10.0.0.255 scope global br-ex\   	valid_lft forever preferred_lft forever
+...
+----
++
+This example shows that controller-0 is exposing the services using the external network, which is not present in
+the {Ceph} nodes, and you need to propagate it through {OpenStackPreviousInstaller}.
+//kgilliga: Can you confirm that this sentence is accurate? ^
```

Comment:
Yes, the example above shows that RGW is exposed through HAPROXY using `10.0.0.103:8080`, and 10.0.0.0/24 is the "external" network, provided by br-ex.